sdk/loadtestservice/load-testing-rest/samples-dev/sample.ts

```diff
@@ -47,28 +50,33 @@ async function main() {
     throw new Error("Test ID returned as undefined.");
 
   // Uploading .jmx file to a test
-  const fileUploadResult = await client
-    .path("/loadtests/{testId}/files/{fileId}", testCreationResult.body.testId, fileId)
-    .put({
-      contentType: "multipart/form-data",
-      body: {
-        file: readStream,
-      },
-    });
+  const fileUploadPoller = await beginUploadTestFile(client, testId, "sample.jmx", {
+    queryParameters: {
+      fileType: "JMX_FILE",
+    },
+    contentType: "application/octet-stream",
+    body: readStream,
+  });
+  const fileUploadResult = await fileUploadPoller.pollUntilDone({
+    abortSignal: AbortController.timeout(60000), // timeout of 60 seconds
+  });
 
-  if (isUnexpected(fileUploadResult)) {
-    throw fileUploadResult.body.error;
-  }
+  if (fileUploadPoller.getOperationState().status != "succeeded")
+    throw new Error(
+      "There is some issue in validation, please make sure uploaded file is a valid JMX." +
+        fileUploadResult
+    );
+
+  console.log(fileUploadResult);
```

Comment:
Should we print more rich information here? Is `fileUploadResult` a string?

Reply:
The fileUploadResult is not useful for user, instead file validation status is useful. Removed this.